mne_bids/mne_bids.py

```diff
@@ -297,9 +346,18 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
     else:
         acq_time = 'n/a'
 
-    df = pd.DataFrame(data={'filename': ['%s' % raw_fname],
-                            'acq_time': [acq_time]},
-                      columns=['filename', 'acq_time'])
+    # append any new scan data to the file
+    if os.path.exists(fname) and write_mode == 'append':
+        df = pd.read_csv(fname, sep='\t')
+        df = df.append(pd.DataFrame(data={'filename': ['%s' % raw_fname],
+                                          'acq_time': [acq_time]},
+                                    columns=['filename', 'acq_time']))
+        df.drop_duplicates(subset='filename', keep='last', inplace=True)
+        df = df.sort_values(by='acq_time')
+    else:
```

Comment:
similar to above: add a short comment that the following stuff will happen if write_mode=='clear' ... only to increase code reading speed for new people

Reply:
same as above but now with `write_mode == 'overwrite'`